react-app/src/components/ProposalDetailScreen/ProposalHeader.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import cn from "classnames";
+import { Proposal } from "../../generated/graphql";
+import Paper from "../common/Paper/Paper";
+import Badge from "../common/Badge/Badge";
+import AppButton from "../common/Buttons/AppButton";
+import LocalizedText from "../common/Localized/LocalizedText";
+import { truncateAddress } from "../../utils/address";
+import UTCDatetime from "../common/DateTime/UTCDatetime";
+
+const ProposalHeader: React.FC<{ proposal: Proposal }> = ({ proposal }) => {
+  const {
+    id,
+    title,
+    status,
+    votingStartTime,
+    votingEndTime,
+    type,
+    proposerAddress,
+    submitTime,
+  } = proposal;
+
+  return (
+    <Paper>
+      <div className={cn("flex", "flex-col", "gap-x-2.5", "items-center")}>
+        <body className={cn("text-xs", "mb-4")}>#{id}</body>
+        <h1
+          className={cn(
+            "text-3xl",
+            "leading-none",
+            "text-center",
+            "text-likecoin-green"
+          )}
+        >
+          {title}
+        </h1>
+      </div>
+      <div className={cn("flex", "flex-col", "my-4", "items-center")}>
+        <Badge variant="light">{status}</Badge>
+      </div>
+      <div
+        className={cn(
+          "flex",
+          "flex-col",
+          "items-center",
+          "justify-around",
+          "my-4",
+          "p-2",
+          "sm:flex-row",
+          "sm:items-start"
+        )}
+      >
+        <div
+          className={cn(
+            "flex",
+            "flex-col",
+            "items-center",
+            "justify-center",
+            "grow",
+            "text-sm"
+          )}
+        >
+          <LocalizedText messageID="ProposalDetail.votingPeriod" />
+          <p className={cn("mb-1", "text-sm", "text-center")}>
+            <UTCDatetime date={votingStartTime} /> to{" "}
```

Comment:
I believe `to` should be localized. 

I think you should define a message for "start date to end date" and pass formatted date string to is arg message argument.